client/my-sites/checkout/checkout-thank-you/redesign-v2/utils.ts

```diff
@@ -32,7 +32,7 @@ export const isRedesignV2 = ( props: CheckoutThankYouCombinedProps ) => {
 
 export function shouldShowConfettiExplosion( purchases: ReceiptPurchase[] ) {
 	if ( isBulkDomainTransfer( purchases ) ) {
```

Comment:
We should probably have just removed this whole conditional block.

Reply:
PR: https://github.com/Automattic/wp-calypso/pull/80064